sdk/dotnet/Compute/InterconnectAttachment.cs

```diff
@@ -490,7 +490,7 @@ public InputList<string> CandidateSubnets
         /// this interconnect attachment. Currently, only 1440 and 1500 are allowed. If not specified, the value will default to 1440.
         /// </summary>
         [Input("mtu")]
-        public Input<string>? Mtu { get; set; }
+        public Input<int>? Mtu { get; set; }
```

Comment:
These changes are breaking. Should we revert in the fork? Or does it fix a bug?

Reply:
From their CHANGELOG:

> compute: corrected underlying type to integer for field mtu in google_compute_interconnect_attachment

Reply:
Was it working at all before? If so, is that a non-breaking change for them but a breaking change for us?

Reply:
From what I can see it's a pure bug fix - it wasn't handled correctly in their code but now it is

Reply:
Okay then